src/libs/Navigation/AppNavigator/NavigationReadyDetector.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Navigation from '../Navigation';
+
+class NavigationReadyDetector extends React.Component {
+    componentDidUpdate() {
+        Navigation.setIsNavigationReady();
+    }
+
+    render() {
+        return null;
```

Comment:
Should this be a `<FullScreenLoadingIndicator />` instead?

Reply:
I don't think that should be included in the responsibilities of this component. We can handle the loading elsewhere. Also, since I'm rendering it in the AppNavigator I don't think that would work well.

Reply:
Okay. Did you create this component as a workaround for [this bug](https://github.com/react-navigation/react-navigation/issues/9170#issuecomment-898831026)?

Reply:
Yes and following @tgolen's guidance from [this review comment](https://github.com/Expensify/App/pull/8855#discussion_r868517039).